pyodide_build/buildpkg.py

```diff
@@ -88,7 +85,7 @@ def compile(path, srcpath, pkg, args):
     try:
         subprocess.run([
             str(Path(args.host) / 'bin' / 'python3'),
-            str(ROOTDIR / 'pywasmcross'),
```

Comment:
should we make this a call to `bin/pyodide`?

Reply:
We can't make it a call to `bin/pyodide` in an obvious way (at least I haven't found one), because `bin/pyodide` is a shell script that will use the python from the current environment by default, while here we want to use the host Python.

Using `host/bin/python -m pyodide_build pywasmcross` may be on some level lighter, because at this point `PYTHONPATH` was already set, so when we start this subprocess we an use `pyodide_build` directly, there is not need to re-set it in `bin/pyodide`..

Reply:
That's a good point.  Thanks.